Diagnostic/rsyslog5/omazurelinuxmds.conf

```diff
@@ -25,4 +25,4 @@ $ActionQueueSaveOnShutdown on
 $ActionQueueFileName lad_mdsd_queue
 $ActionQueueDiscardSeverity 8
 local6.* :omazuremds:;fmtfile
-& ~
+& stop
```

Comment:
Hi @hosungsmsft alright, yes, I wanted to see exactly what those config sections mean too , but I would probably change the filenames just to be safe ;) thanks for the updates.